### DISCUSSION TITLE
如興掏空案 坑殺國發基金

### DISCUSSION THREAD

**Discussion by Floriaand** [1101-GS4719-C/Python-Community-GS4719-C#702] [General]
我認為金管會在這次的事件中要承擔大部分的責任，且他們通過如興現金增資案的時機很讓人多想，三次退件卻在蔡英文拜訪後通過，後來如興開始被掏空後，員工爆料，金管會也反應慢半拍，最後讓如興董事長成功跑路，錢也沒了
網址: https://www.businessweekly.com.tw/magazine/Article_page.aspx?id=6008099

**Comment by CPCpyt** [comment]
雖然引發這樣的醜聞，但我想幕後的策畫也從中獲利不少。
看到網上的哀號我想就是貪，進入本身就高投機的投資標的。
